ext/standard/array.c

```diff
@@ -2553,7 +2553,7 @@ PHP_FUNCTION(array_fill)
 
 	if (EXPECTED(num > 0)) {
 		if (sizeof(num) > 4 && UNEXPECTED(EXPECTED(num > 0x7fffffff))) {
-			zend_value_error("Too many elements");
+			zend_argument_value_error(2, "is too big");
```

Comment:
Use ``large`` maybe rather than ``big`` ?

Reply:
Yes! I knew that something is not right with it, but didn't have the brainower yesterday.. :D